ccd-definition/AuthorisationCaseField/CareSupervision/la-barrister.json

```diff
@@ -856,10 +863,17 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "manageDocumentsCourtBundle",
+    "CaseFieldID": "manageDocumentsCourtBundleHearingLabel",
     "UserRole": "[LABARRISTER]",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "manageDocumentsCourtBundle",
+    "UserRole": "[LABARRISTER]",
+    "CRUD": "CRUD"
+  },
   {
```

Comment:
Double check whether they need access to as LA?

Reply:
This is aligning the authorisation settings of confidentialRespondents, confidentialOthers and confidentialChildren, which affect the visibility of "Confidential Information" tab.

Double-check with Feisal